docs/ref/contrib/gis/functions.txt

```diff
@@ -230,13 +230,18 @@ south = ``π``; west = ``3π/2``.
 
 *Availability*: `PostGIS <https://postgis.net/docs/ST_MinimumBoundingCircle.html>`__,
 `Oracle <https://docs.oracle.com/en/database/oracle/oracle-database/21/spatl/
-SDO_GEOM-reference.html#GUID-82A61626-BB64-4793-B53D-A0DBEC91831A>`_
+SDO_GEOM-reference.html#GUID-82A61626-BB64-4793-B53D-A0DBEC91831A>`_, 
+`SpatiaLite 5.1+ <https://www.gaia-gis.it/gaia-sins/spatialite-sql-5.1.0.html>`_.
```

Comment:
There is no need for a reference if we don't have an anchor for the `GEOSMinimumBoundingCircle`.